pkg/monitoring/metrics/infrastructure_metrics.go

```diff
@@ -0,0 +1,41 @@
+package metrics
+
+import (
+	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+	ioprometheusclient "github.com/prometheus/client_model/go"
```

Comment:
What does `io` stand for in `ioprometheusclient` here?

Reply:
ioprometheusclient is a client library used for interacting with Prometheus, focusing on "input/output" (io) operations.